hardware/raspberrypi/usb/README.md

```diff
@@ -2,13 +2,17 @@
 
 ## Page Contents
 
-- [Overview for devices prior to the Pi 4](#overview)  
-- [Overview for the Pi 4](#overview_pi4)   
-- [Supported devices](#support)
-- [General limitations (not Pi 4)](#genlimits)
-- [Port power limits](#powerlimits)
-- [Known issues (not Pi 4)](#knownissues)
-- [Troubleshooting](#troubleshooting)
+- [USB](#usb)
+  - [Page Contents](#page-contents)
```

Comment:
Not sure these two need to be in the list themselves.

Reply:
I don't know where that change has come from - I didn't do it!

Reply:
Weird again. Looking at this I still have no idea where those changes have come from. I'll manually remove them.